concepts/nil/introduction.md

```diff
@@ -1,19 +1,67 @@
 # Introduction
```

Comment:
It looks like this file is just a copy of the `about.md` document, which is not exactly how it should be (see [the docs](https://exercism.org/docs/building/tracks/concept-exercises)). In short: the introduction should be a minimal explanation of the concept.

Reply:
Updated!